lib/src/client.dart

```diff
@@ -140,3 +141,48 @@ abstract class Client {
   /// do so can cause the Dart process to hang.
   void close();
 }
+
+const _clientToken = Object();
+
+/// Return the [Client] for the current [Zone], if one has been set.
+///
+/// NOTE: This function is explicitly hidden from the public API.
+Client? getZonedClient() => Zone.current[_clientToken] != null
```

Comment:
```suggestion
Client? get zoneClient => Zone.current[_clientToken] != null
```
https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

Reply:
Done.